src/modules/questionnaire/questionnaire.module.ts

```diff
@@ -7,6 +7,7 @@ import { EffectsModule } from '@ngrx/effects';
 import { questionnaireReducer } from './state/questionnaire.reducer';
 import { QuestionnaireEffects } from './state/questionnaire.effects';
 import { QuestionnaireRouterModule } from './questionnaire.router.module';
+import { SERVICES } from '../questionnaire/services/index';
```

Comment:
@jorgevillalobosGL `/index` is not required since typescript will load the index.ts when route is a path.